frontend/src/metabase/entities/dashboards.js

```diff
@@ -1,6 +1,10 @@
 import { t } from "ttag";
 
-import { automagicDashboardsApi, dashboardApi } from "metabase/api";
+import {
```

Comment:
This change was supposed to be merged in #50623 but got lost in the pile of sub-PRs.